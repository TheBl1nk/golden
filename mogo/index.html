<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Document</title>
</head>
<body>
    <header>
        <a href="">
            <img src="img/MoGo.png" alt="">
        </a>
        <!-- nav>a*5 -->
        <nav>
            <a href="">ABOUT</a>
            <a href="">SERVICE</a>
            <a href="">WORK</a>
            <a href="">BLOG</a>
            <a href="">CONTACT</a>
        </nav>
    </header>
    <main>
        <section>
            <div>
                <h3>We work with</h3>
                <h2>Amazing Services</h2>
                <div class="line"></div>
            </div>
            <ul>
                <li>
                    <img src="img/alarm 1.png" alt="alarm">
                    <h4>photography</h4>
                    <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor.</p>
                </li>
                <li>
                    <img src="img/bar-graph 1.png" alt="bar-graph">
                    <h4>photography</h4>
                    <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor.</p>
                </li>
                <li>
                    <img src="img/display 1.png" alt="display">
                    <h4>photography</h4>
                    <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor.</p>
                </li>
                <li>
                    <img src="img/paper 1.png" alt="paper">
                    <h4>photography</h4>
                    <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor.</p>
                </li>
                <li>
                    <img src="img/home 1.png" alt="home">
                    <h4>photography</h4>
                    <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor.</p>
                </li>
                <li>
                    <img src="img/digital-marketing 1.png" alt="digital-marketing">
                    <h4>photography</h4>
                    <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor.</p>
                </li>
            </ul>
        </section>

        <section>
            <div>
                <h3>Who we are</h3>
                <h2>Meet our team</h2>
                <div class="line"></div>
                <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. </p>
            </div>
            <ul>
                <li>
                    <figure>
                        <img src="img/first-man 1.png" alt="">
                        <figcaption>
                            <h4>Matthew Dix</h4>
                            <p>Graphic Design</p>
                        </figcaption>
                    </figure>
                </li>
                <li>
                    <figure>
                        <img src="img/second-man 1.png" alt="">
                        <figcaption>
                            <h4>Christopher Campbell</h4>
                            <p>Branding/UX design</p>
                        </figcaption>
                    </figure>
                </li>
                <li>
                    <figure>
                        <img src="img/third-man 1.png" alt="">
                        <figcaption>
                            <h4>Michael Fertig </h4>
                            <p>Developer</p>
                        </figcaption>
                    </figure>
                </li>
            </ul>
        </section>
    </main>

    <footer>
        <p>Â© 2016 MoGo free PSD template by Laaqiq</p>
        <form action="">
            <input type="email" placeholder="Your Email...">
            <button type="submit">Subscribe</button>
        </form>
    </footer>
</body>
</html>