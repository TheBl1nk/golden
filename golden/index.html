<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Document</title>
</head>
<body>
    <header>
        <a href="#">
            <img src="./img/Logo.png" alt="logo">
        </a>

        <!-- nav>a*5 -->
        <nav> 
            <a href="">Home</a>
            <a href="">Services</a>
            <a href="">Portfolio</a>
            <a href="">About</a>
            <a href="">About</a>
        </nav>

    </header>

    <main>
        <section>
            <h2>Services</h2>
            <ul>
                <li>
                    <img src="img/Group 3.png" alt="">
                    <h3>E-Commerce</h3>
                    <p>Proin iaculis purus consequat sem cure  digni ssim. Donec porttitora entum suscipit  aenean rhoncus posuere odio in tincidunt.</p>
                </li>
                <li>
                    <img src="img/Group 1.png" alt="">
                    <h3>Responsive Web</h3>
                    <p>Proin iaculis purus consequat sem cure  digni ssim. Donec porttitora entum suscipit  aenean rhoncus posuere odio in tincidunt.</p>
                </li>
                <li>
                    <img src="img/Group 2.png" alt="">
                    <h3>Web Security</h3>
                    <p>Proin iaculis purus consequat sem cure  digni ssim. Donec porttitora entum suscipit  aenean rhoncus posuere odio in tincidunt.</p>
                </li>
            </ul>
        </section>
        <section>
            <h2>Contact us</h2>

            <form action="">
                <div>
                    <label for="">name</label><br>
                    <input type="text" id="firstname" required placeholder="your name*">
                    <br><br>
                    <label for="">email</label><br>
                    <input type="email" id="email" required placeholder="your e-mail*">
                    <br><br>
                    <label for="text">subject</label><br>
                    <input type="text" id="" placeholder="subject">
                    <br><br>
                </div>
                <div>
                    <label for="comment">comment</label><br>
                    <textarea name="comment" id="comment" required placeholder="your message*"></textarea>
                </div>
                <button type="submit">Send Message</button>
            </form>
        </section>
    </main>
</body>
</html>